gateway-service/build.gradle

```diff
@@ -34,8 +34,8 @@ gitProperties {
 dependencies {
     compile(project(':common-service-core'))
     compile(project(':apiml-common'))
+    compile(project(':apiml-security-common'))
```

Comment:
Same situation that was in apiml-security-common. Dependencies: apiml-security-common includes apiml-common, which includes common-service-core.